core/players-handbook-2024/spells.xml

```diff
@@ -0,0 +1,8977 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<elements>
```

Comment:
Did a full review of the spells file, and found a bunch of typos in need of correcting in both of our sections. Would be over 100 different comments here, so I'm just uploading the file itself and you can review and merge it. 

[spells.zip](https://github.com/user-attachments/files/17987405/spells.zip)